### ISSUE TITLE
Symbols with dot did not work always

### ISSUE THREAD

**Issue by MdmrBerlin** [kirlut/AlphaVantage.Net#25]
BMW.FRK is an valid symbol for example 
This is not working in Demo Core

`var querya = new Dictionary<string, string>()
            {
                {"symbol", "BMW.FRK"} 
            };
            string test = await client.RequestPureJsonAsync(ApiFunction.OVERVIEW, querya,true);`

But this works fine
` StockTimeSeries stockTs = await stocksClient.GetTimeSeriesAsync("BMW.DEX", Interval.Daily, OutputSize.Compact, isAdjusted: true);

            GlobalQuote globalQuote = await stocksClient.GetGlobalQuoteAsync("BMW.DEX");
`

Is there any howto handle symbols with dot ? Most of non US Symbols has an dot.

**Comment by kirlut** [comment]
@MdmrBerlin I don't think that they are supported by API itself. For example, call https://www.alphavantage.co/query?function=OVERVIEW&symbol=BMW.DEX&apikey=1 returns empty response
